pkg/gameservers/controller.go

```diff
@@ -316,6 +320,44 @@ func (ext *Extensions) creationValidationHandler(review admissionv1.AdmissionRev
 	return review, nil
 }
 
+// creationMutationHandlerPod that mutates a GameServer pod when it is created
+// Should only be called on gameserver pod create operations.
+func (ext *Extensions) creationMutationHandlerPod(review admissionv1.AdmissionReview) (admissionv1.AdmissionReview, error) {
+	logger := runtime.NewLoggerWithSource("gke")
```

Comment:
The `Extensions` type already has a `baseLogger` in it. Personally since you only use it once, I would just remove this variable and on line 336, I would do `ext.baseLogger.WithField(...)`

That said, logging the whole review object is probably excessive even in Debug. I'd log the Pod name / namespace so you can track the flow as necessary, though.